plugins/modules/iam_user.py

```diff
@@ -41,6 +41,16 @@
     default: false
     type: bool
     aliases: ['purge_policy', 'purge_managed_policies']
+  tags:
```

Comment:
version_added is missing. The next will be 2.0.0 I think.

Reply:
for `tags` and also for `purge_tags`

Reply:
I've added `version_added: 2.0.0` to both tags.